### PR TITLE
ACS-6707: Add CamelNodePublisher integration test

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/TestCamelConsumer.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        if (messagesCount != messages.size())
+        {
+            Thread.sleep(DELAY);
+        }
+
+        assertEquals(messages.size(), messagesCount);
+    }
+
+    @SneakyThrows
+    public void asertMessageReceived(String message)
```
Comment (krdabrowski) [inline bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/TestCamelConsumer.java:83]: Typo - missing "s" in `asertMessageReceived`

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/ActiveMqIntegrationTestBase.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    private static final String ACTIVE_MQ_TAG = DockerTags.getOrDefault("activemq.tag", "5.18.3-jre17-rockylinux8");
+    private static final int ACTIVE_MQ_PORT = 61616;
+    private static final String BULK_INGESTER_QUEUE = "test.bulk.ingester.queue";
+
+    @Container
+    static GenericContainer<?> activemq = createAMQContainer();
+
+    public static GenericContainer<?> createAMQContainer()
+    {
+        return new GenericContainer(DockerImageName.parse(ACTIVE_MQ_IMAGE).withTag(ACTIVE_MQ_TAG))
```
Comment (krdabrowski) [inline bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/ActiveMqIntegrationTestBase.java:52]:
```suggestion
        return new GenericContainer<>(DockerImageName.parse(ACTIVE_MQ_IMAGE).withTag(ACTIVE_MQ_TAG))
```

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/TestCamelConsumer.java (hunk 1 above)
Comment (Pawel-608) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/TestCamelConsumer.java]: fixed!

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/ActiveMqIntegrationTestBase.java (hunk 2 above)
Comment (Pawel-608) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/ActiveMqIntegrationTestBase.java]: fixed!

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/DockerTags.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+
+/**
+ * Allows to access properties from 'docker-tags.properties' file. It's required to run: `mvn package` before accessing the properties file, to allow Maven filter out and replace variables.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class DockerTags
```
Comment (krdabrowski) [inline bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/DockerTags.java:41]: Please add here also `DockerTagsIntegrationTest` to confirm that on CI everything works fine

Comment (Pawel-608) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/util/DockerTags.java:43]: added

**Review by krdabrowski — APPROVED** [review]
LGTM

**Pull request merged**